File: java/Problem10.java
import java.util.*;
import java.io.*;

public class Main { 
  

    static boolean checkPrime(int numberToCheck) 
    { 
        if(numberToCheck == 1) { 
            return false; 
        } 
        for (int i = 2; i*i <= numberToCheck; i++) { 
            if (numberToCheck % i == 0) { 
                return false; 
            } 
        } 
        return true; 
    } 
  

    static int primeSum(int l, int r) 
    { 
        int sum = 0; 
        for (int i = r; i >= l; i--) { 
  
            // Check for prime 
            boolean isPrime = checkPrime(i); 
            if (isPrime) { 
  
                // Sum the prime number 
                sum = sum + i; 
            } 
        } 
        return sum; 
    } 

    public static void main(String[] args) 
    { 
        Scanner sc=new Scanner(System.in);
        int l = 0;
        int r=sc.nextInt();
  
        // Call the method with l and r 
        System.out.println("The sum of the primes below "+r+" is "+primeSum(l, r)); 
    } 
} 

File: python/problem10.py
#!/usr/bin/env python

"""  Provides the sum of all the primes below two million"""

def is_prime(num):
  """is_prime checks for the number "num"
   if the number "num" is prime  it returns true 
   if not prime returns false"""

  if (num<=1):
     return False
  for i in range(2,int (num**0.5)+1):
        if (num%i == 0):
            return False
  return True
#initialize  variable "sum" 
sum = 0
#iterate the loop from 2 to 2 million
for i in range(2,2000000):
 #check if num is prime 
    if ( is_prime(i)):
 # If num is prime , Add num to variable sum 
        sum = sum + i
#print value of sum
print(sum)
